pkg/kn/commands/service/create_test.go

```diff
@@ -134,6 +134,12 @@ func TestServiceCreateImage(t *testing.T) {
 	}
 }
 
+func TestServiceCreateWithMultipleImages(t *testing.T) {
+	_, _, _, err := fakeServiceCreate([]string{
+		"service", "create", "foo", "--image", "gcr.io/foo/bar:baz", "--image", "gcr.io/bar/foo:baz", "--no-wait"}, false, false)
+	assert.Error(t, err, "invalid argument \"gcr.io/bar/foo:baz\" for \"--image\" flag: can be provided only once")
```

Comment:
What we also do typically in unit test to assert on the content of the error message: not the exact wording but that key context information is included. This helps us to ensure that the error message is meaningful, but the test is also as robust as possible so that the exact wording could be changed without breaking the tests.

We have an utility `util.ContainsAll()` which helps here. Check for other tests that use that method.